web/package/cockpit-agama.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Nov 30 15:19:38 UTC 2023 - José Iván López González <jlopez@suse.com>
+
+- Allow selecting the storage policy to make free space
```

Comment:
maybe I would rephrase it as `Allow selecting the storage policy to make space for installation`